torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1480,17 +1480,6 @@ def call_function(
 
 
 class TemplatedAttentionHigherOrderVariable(TorchHigherOrderOperatorVariable):
-    @staticmethod
-    def normalize_to_args(args, kwargs):
-        # input signature is (query, key, value, score_mod, *other_buffers)
-        # Flatten args and kwargs into lists
-        flat_args = pytree.tree_flatten(args)[0]
-        flat_kwargs = pytree.tree_flatten(kwargs)[0]
-
-        # Combine the flattened lists
-        all_args = flat_args + flat_kwargs
-        return all_args
```

Comment:
Since we intend to make ```block_mask``` as a tuple, so we should not normalize it.

Reply:
hmm... we still need to normalize the kwargs into args.

Reply:
Updated to keep kwargs normalization.

Reply:
tbh I'm not sure if this is right at all.... but I'm also not sure if it matters? Since users aren't directly calling this API, and I think the normalization occurs at a higher level?